cmd/brew-upgrade-mysql

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+update_my_cnf() {
+  CNF_PATH="$(brew --prefix)/etc/my.cnf"
+
+  echo "Backing up old my.cnf file..."
+  cp -vf $CNF_PATH "$CNF_PATH.github"
+
+  cat > $CNF_PATH <<-EOM
+# For advice on how to change settings please see
+# http://dev.mysql.com/doc/refman/5.7/en/server-configuration-defaults.html
+
+[mysqld]
+
+innodb_strict_mode=OFF
+optimizer_switch='index_merge_intersection=OFF'
+query_cache_size=0
+sql_mode=NO_ENGINE_SUBSTITUTION
+EOM
+}
+
+set +e
+brew ls mysql@5.6 &>/dev/null
+MYSQL_5_6="$?"
+
+brew ls mysql@5.7 &>/dev/null
+MYSQL_5_7="$?"
+set -e
+
+if [[ $MYSQL_5_6 == "0" || $MYSQL_5_7 == "0" ]]; then
+  if [[ $MYSQL_5_6 == "0" ]]; then
+    set +e
+    brew services stop mysql@5.6
+    set -e
```

Comment:
Nitpick, but the pattern I usually use when I've `set -e` but a command might fail is to use `|| true`, e.g. `brew services stop mysql@5.6 || true`